File: v2/csharp/src/main/csharp/vestorly/Model/Eventcreateresponse.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace vestorly.model {
  [DataContract]
  public class Eventcreateresponse {
    
    
    [DataMember(Name="meta", EmitDefaultValue=false)]
    public Meta meta { get; set; }

    
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Eventcreateresponse {\n");
      
      sb.Append("  meta: ").Append(meta).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/java/src/main/java/vestorly/client/model/Eventcreateresponse.java
package vestorly.client.model;

import vestorly.client.model.Meta;

import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Eventcreateresponse  {
  
  private Meta meta = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("meta")
  public Meta getMeta() {
    return meta;
  }
  public void setMeta(Meta meta) {
    this.meta = meta;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Eventcreateresponse {\n");
    
    sb.append("  meta: ").append(meta).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
